snuba/query/columns.py

```diff
@@ -54,6 +55,8 @@ def column_expr(
         return complex_column_expr(
             dataset, [aggregate, column_name, alias], query, parsing_context
         )
+    elif isinstance(column_name, str) and is_datetime(column_name):
+        return escape_literal(is_datetime(column_name))
```

Comment:
Which use case are we talking about here ? I am not sure it would produce what you want since clickhouse needs a datetime string to be provided as `toDateTime(string)`

Reply:
`escape_literal` formats the value with `toDateTime`, if the input is a datetime.

Is that what you're asking about?

Reply:
oh right, ..... I have a hard time containing my disappointment, ... ehm ... relief in remembering that detail about escape_literal